src/pages/creer-un-jeu/mimique/4.tsx

```diff
@@ -20,6 +21,13 @@ const MimiqueStep4 = () => {
   const [isLoading, setIsLoading] = React.useState(false);
   const data = (activity?.data as MimicsData) || null;
 
+  const isValid = React.useMemo(() => {
+    if (activity !== null && activity.content.filter((c) => c.value.length > 0 && c.value !== '<p></p>\n').length === 0) {
+      return false;
+    }
+    return true;
+  }, [activity]);
```

Comment:
And again here but these time with the 3 steps:
```tsx
const errorSteps = React.useMemo(() => {
    const errors: number[] = [];
    if (!isMimicValid(data.game1)) {
         errors.push(0); // step of mimic 1
    }
    if (!isMimicValid(data.game2)) {
         errors.push(1); // step of mimic 2
    }
    if (!isMimicValid(data.game3)) {
         errors.push(2); // step of mimic 3
    }
    return errors;
  }, [data]);
const isValid = errorSteps.length === 0;
```

Reply:
I made the change here too as suggested. I changed also the logical in the files to adapt the cards regarding the validation or not. It seems to work.